db/comdb2.c

```diff
@@ -5715,8 +5715,7 @@ static struct dbenv *newdbenv(char *dbname, char *lrlname)
         return NULL;
     }
 
-    if (lrlname)
-        pre_read_lrl_file(dbenv, lrlname, dbname);
+    if (lrlname) pre_read_lrl_file(dbenv, lrlname, dbname);
```

Comment:
Yes, I would like that, it is easier to set a breakpoint on condition match if two lines. Change roborivs?